.gitignore

```diff
@@ -43,4 +43,7 @@ version.h
 Makefile.dep
 make_config.mk
 .vscode
-kvrocks2redis
\ No newline at end of file
+kvrocks2redis
+
+src/compactdb/
+src/testsdb/
```

Comment:
```suggestion
```
why did you add these, to avoid your tests polluting git repo? i think it is not necessary if just because of tests

Reply:
Those are default test database pathes when I run on kvrocks project path, I guess other developer will generate those test databases on `kvrocks/src` directory, so I ignore those pathes for developer convenience.